### PR TITLE
Implement commit-tool, get-page-count and handle-back-button

### PR DESCRIPTION
**Pull request by Maxwell-Yang-PDFTron**

### CODE REVIEW

File: android/src/main/java/com/pdftron/pdftronflutter/PluginUtils.java [hunk 1]
```diff
@@ -259,6 +265,25 @@ public static void onMethodCall(MethodCall call, MethodChannel.Result result, Vi
...
+                    getPageCount(result, component);
+                } catch (PDFNetException ex) {
+                    ex.printStackTrace();
+                    result.error(Long.toString(ex.getErrorCode()), "PDFTronException Error: " + ex, null);
+                }
+                break;
+            }
+            case FUNCTION_HANDLE_BACK_BUTTON: {
+                checkFunctionPrecondition(component);
+                handleBackButton(result, component);
```
Comment (sgong-pdftron) [inline android/src/main/java/com/pdftron/pdftronflutter/PluginUtils.java:285]: missing break

File: README.md [hunk 2]
```diff
@@ -291,6 +291,38 @@ Saves the currently opened document in the viewer and returns the absolute path
...
 ```
 
+### PdftronFlutter.commitTool()
+
+Commits the current tool.
+
+Only available for multi-stroke ink and poly-shape, and will return false for all other tools.
+
+```dart
+PdftronFlutter.commitTool().then((value) {
```
[the remark this answers is not in the document]
Comment (sgong-pdftron) [reply README.md:329]: for all three

**Review by sgong-pdftron — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -319,6 +319,35 @@ Saves the currently opened document in the viewer and returns the absolute path
 var path = await PdftronFlutter.saveDocument();
 ```
 
+### PdftronFlutter.commitTool()
+
+Commits the current tool.
```
Comment (JamesTron) [inline README.md:324]: Maybe instead:
Commits the annotation being created by the tool to the PDF.

File: README.md [hunk 4]
```diff
@@ -319,6 +319,35 @@ Saves the currently opened document in the viewer and returns the absolute path
...
+Only available for multi-stroke ink and poly-shape, and will return false for all other tools.
+
+```dart
+var committed = await PdftronFlutter.commitTool();
+print("Tool committed: $committed");
+```
+
+### PdftronFlutter.getPageCount()
+
+To get the current page count of the document.
```
Comment (JamesTron) [inline README.md:335]: Instead:
Returns the total number of pages in the currently displayed document.

**Review by JamesTron — CHANGES_REQUESTED** [review]
Just some documentation suggestions.

**Review by JamesTron — APPROVED** [review]

**Review by sgong-pdftron — APPROVED** [review]

**Pull request merged**